### ISSUE TITLE
Memories Download

### ISSUE THREAD

**Issue by bicepjai** [icloud-photos-downloader/icloud_photos_downloader#684]
## Summary
Ability to download photo generated memories videos/slideshow with music

## Context
If we delete media from icloud say because the icloud is full, there is no way to get the memories generated in the photo app. Ability to download the memories will be monumental.

**Comment by AndreyNikiforov** [comment]
Are the memories auto generated by Apple? Can you download these memories after deleting originals from icloud.com website? I would assume that auto generated content will be refreshed once originals are deleted.

**Comment by bicepjai** [comment]
yeah, they are auto generated and not auto downloadable. I ended up spending time downloading manually into an album and let that sync in.

**Comment by AndreyNikiforov** [comment]
Where do you see auto generated memories? Are they shown in icloud.com (though browser)? Are they shown as album? If they are, can you try `icloudpd --list-albums` command to see if they are displayed and then try to download them as album?

**Comment by fjbravo** [comment]
Memories can be seen on icloud.com, but don't show up with icloudpb --list-albums.
<img width="265" alt="image" src="https://github.com/user-attachments/assets/39708854-a4a1-4d05-87f6-1e4a72c6a1f2" />
